ambassador/tests/t_lua_scripts.py

```diff
@@ -20,7 +25,7 @@ def manifests(self) -> str:
   config:
     lua_scripts: |
       function envoy_on_response(response_handle)
-        response_handle: headers():add("Lua-Scripts-Enabled", "Processed")
+        response_handle: headers():add("Lua-Scripts-Enabled", "$LUA_SCRIPTS_ENABLED")
```

Comment:
How about swap one of these to be the `${}` form so we have coverage of that syntax too?

Reply:
I actually had that originally but landed on a KeyError as format was picking it up, I guess I could pass the right args to format to not blow up, let me see.

Reply:
Ok easy I made the other test use the ${} style, no need to mess with anything!